redirects-transition.conf

```diff
@@ -461,9 +668,13 @@ rewrite ^/disclosurehs/(.*) https://www.fec.gov/data/browse-data/?tab=candidates
 rewrite ^/disclosureie/(.*) https://www.fec.gov/data/independent-expenditures/?most_recent=true&data_type=processed&is_notice=true redirect;
 rewrite ^/disclosurep/(.*) https://www.fec.gov/data/candidates/president/ redirect;
 
+# elecfil/ broader redirects
+rewrite ^/elecfil/authorized_manual/(.*) https://efilingapps.fec.gov/fecfiledoc/fecfiledoc.pdf redirect;
+rewrite ^/elecfil/formats/(.*) https://efilingapps.fec.gov/registration/softwarelogs.htm redirect;
+
 # fecig/ broader redirects
-rewrite "^/fecig/(.*)" https://www.fec.gov/resources/cms-content/documents/$1;
-rewrite "^/fecig/documents/(.*)" https://www.fec.gov/resources/cms-content/documents/$1;
+rewrite "^/fecig/(.*)" https://www.fec.gov/resources/cms-content/documents/$1; redirect;
```

Comment:
We'll want this to redirect to the OIG homepage I think: https://www.fec.gov/office-inspector-general/. Since you already have a redirect for the documents below.

Reply:
Updated. Thanks for catching.